src/hooks/useGetOrders.tsx

```diff
@@ -149,6 +152,43 @@ export function useGetTxOrders(txHash: string): GetTxOrdersResult {
   return { orders, error, isLoading: isLoading || areErc20Loading, errorTxPresentInNetworkId }
 }
 
+export function useTxOrderExplorerLink(
+  txHash: string,
+  orders: Order[] | undefined,
+): ExplorerLinkProps | Record<string, unknown> {
+  const networkId = useNetworkId() || undefined
+  const [explorerLink, setExplorerLink] = useState<ExplorerLinkProps | Record<string, unknown>>({})
+
+  useEffect(() => {
+    if (!networkId) {
+      return
+    }
+
+    if (!orders?.length) {
```

Comment:
Nice solution,

I see that you only need the number of orders here, maybe use something like `isZeroOrders` instead of `orders: Order[]` would win:
1. improve readability with less conditionals added.
```javascript
if (!networkId || !isZeroOrders) return

for (const ...
.
.
.
```
2. Use **primitive** value in the hook.